postcss.json

```diff
@@ -0,0 +1,12 @@
+{
+  "autoprefixer": {
+    "browsers": [
+      "last 2 versions",
+      "safari >= 7",
+      "ie >= 9",
+      "ff >= 20",
+      "ios >=7",
+      "android >=4"
+    ]
+  }
+}
```

Comment:
I think having Postcss is a good tool to have, we could use it for other improvements in CSS